### DISCUSSION TITLE
Installation for windows where snap v10 is installed

### DISCUSSION THREAD

**Discussion by shimonfrancis** [johntruckenbrodt/pyroSAR#327] [General]
I installed pyroSAR in anaconda where my SNAP version is 10 but it reflects some errors . When i try it with SNAP version 9 it works fine. Can some one point me what im missing.

**Comment by johntruckenbrodt** [comment]
Hi @shimonfrancis. Likely you have installed an older version of pyroSAR that does not yet support SNAP 10. The problem is that the latest pyroSAR versions have an upper limit on the Python version used but the older ones don't. So, if you install pyroSAR into an environment with Python>3.10 already installed, an older version will be installed. I understand this is inconvenient.

Further reading: https://github.com/johntruckenbrodt/pyroSAR/issues/315  
A fix is in the making: https://github.com/johntruckenbrodt/pyroSAR/pull/328

**Reply by johntruckenbrodt** [reply]
I have just released version 0.27.0, which should fix this.

**Reply by shimonfrancis** [reply]
Hi @johntruckenbrodt.  Now its working for SNAP version 10 and pyroSAR is a wonderfull package which helps in processing large scale SAR imagery. Thanks for your effort and thanks for the team pyroSAR. Is this new release compatible with SNAP 11?

**Reply by johntruckenbrodt** [reply]
Awesome 😃! I have not tried SNAP 11 yet. You can give it a try, maybe it works. I'd appreciate the feedback.
